### DISCUSSION TITLE
Utility method to generate default formData based on schema ?

### DISCUSSION THREAD

**Discussion by naveen8801** [rjsf-team/react-jsonschema-form#4166] [Q&A]
I have a scenario where I want to generate default formData based on the schema, after some research I found a utility method named `getDefaultFormState` which is working fine for all the attributes present inside the `properties` key of the schema but I've a schema in which `allOf` is present at properties level and for that schema mentioned method is ignoring defaults present inside allOf 

Is there any work around for this ?

**Comment by heath-freenome** [comment]
As far as I can tell, that method does not support `allOf`. Not sure if there is a work-around. sorry
